tests/mlio-py-test/test_csv_reader.py

```diff
@@ -0,0 +1,126 @@
+from collections import defaultdict
```

Comment:
Please make sure to clean all `flake8` warnings for this file.

Reply:
Oops. I don't why I forgot to lint this file. Should be fixed now.